mage_integrations/mage_integrations/destinations/oracledb/__init__.py

```diff
@@ -0,0 +1,216 @@
+import re
+from typing import Dict, List
+
+from mage_integrations.connections.oracledb import OracleDB as OracleDBConnection
+from mage_integrations.destinations.constants import (
+    COLUMN_TYPE_OBJECT,
+    UNIQUE_CONFLICT_METHOD_UPDATE,
+)
+from mage_integrations.destinations.oracledb.utils import (
+    build_alter_table_command,
+    build_create_table_command,
+    clean_column_name,
+    convert_column_type,
+)
+from mage_integrations.destinations.sql.base import Destination, main
+from mage_integrations.destinations.sql.utils import (
+    build_insert_command,
+    column_type_mapping,
+)
+
+
+class OracleDB(Destination):
+    @property
+    def host(self) -> str:
+        return self.config['host']
+
+    @property
+    def port(self) -> str:
+        return self.config['port']
+
+    @property
+    def service(self) -> str:
+        return self.config['service']
+
+    @property
+    def user(self) -> str:
+        return self.config['user']
+
+    @property
+    def password(self) -> str:
+        return self.config['password']
+
+    @property
+    def database(self) -> str:
+        return self.config['database']
+
+    def build_connection(self) -> OracleDBConnection:
+        return OracleDBConnection(
+            host=self.host, password=self.password,
+            user=self.user, port=self.port, service=self.service)
+
+    def build_create_schema_commands(
+        self,
+        database_name: str,
+        schema_name: str,
+    ) -> List[str]:
+        # "create schema" in oracle db has nothing to do with schema creation.
+        # While a "schema" is a user ID and a collection of the users tables and indexes.
+        return [
+            'SELECT name FROM v$database',
+        ]
+
+    def test_connection(self) -> None:
+        conn = self.build_connection().build_connection()
+        cursor = conn.cursor()
+        try:
+            cursor.execute("SELECT name FROM v$database")
+        except Exception as exc:
+            self.logger.error(f"test_connection exception: {exc}")
+            raise exc
+        return
+
+    def build_create_table_commands(
+        self,
+        schema: Dict,
+        schema_name: str,
+        stream: str,
+        table_name: str,
+        database_name: str = None,
+        unique_constraints: List[str] = None,
+    ) -> List[str]:
+        return [
+            build_create_table_command(
+                column_type_mapping=column_type_mapping(
+                    schema,
+                    convert_column_type,
+                    lambda item_type_converted: 'LONG',
```

Comment:
I don't think it works. LONG is used to store number. This is lambda function is to map the array of item_type_converted to a column type.
Can you test syncing an array type?